package.json

```diff
@@ -109,7 +114,16 @@
         "path": "./syntaxes/expression.json",
         "scopeName": "expression.ng"
       }
-    ]
+    ],
+    "menus": {
```

Comment:
Because these items map directly to commands that are defined above, does it make sense to keep the "commands" item and the "menus" item next to each other?

Reply:
sure!